tests/unit/services.py

```diff
@@ -50,4 +50,7 @@ def pdf_url_builder_service(mock_service):
 
 @pytest.fixture
 def jstor_api(mock_service):
-    return mock_service(JSTORAPI)
+    jstor_api = mock_service(JSTORAPI)
+
+    jstor_api.is_jstor_url.return_value = False
```

Comment:
We use this to check if things are JSTOR URLs indirectly in a lot of tests. By default mocks evaluate to true, and therefore _everything_ becomes a JSTOR URL which is likely not the intention of most of the tests.